probers/pidfile/probe.go

```diff
@@ -0,0 +1,48 @@
+package pidfile
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+func (p *pidconfig) probe() error {
+	p.setpidfile(true)
```

Comment:
Why is this line here? This will set a transient "healthy" for an unhealthy service. This should be at the end of the function.